### ISSUE TITLE
Create detailed overview of a challenge

### ISSUE THREAD

**Issue by pbochis** [coduno/app#43]
That page should contain:
- Details of the challenge
- Provide an overview of the tasks defined for that challenge (maybe cards).
- A leaderboard of users who coded that challenge
- [x] Basic layout
- [x] Displaying challenge properties
- [x] Chart on the right top ("Skills chart")
- [ ] Fancy leaderboard
- [x] Task cards

**Comment by pbochis** [comment]
@flowlo : Me and @victorbalan did a mockup of the challenge detail page. What do you think about it?
![challenge-detail-mockup](https://cloud.githubusercontent.com/assets/9076897/9194221/56b5b310-4022-11e5-8f21-85c79a9333d1.jpg)

**Comment by lorenzleutgeb** [comment]
- Duration is missing.
- The top right should by default be the combination of skills from the tasks, and to be switched to the average performance I think. First, to get an overview of what the challenge is about, and second how did candidates really perform.
- Average makes no sense without deviation, so I would also indicate this on the graph.

WDYT?

**Comment by victorbalan** [comment]
This is a start mockup that we thought about. We have not included the task list anywhere for now. Do you have any idea where that could fit?
The idea with combination of skills from the tasks is good, I like it.

**Comment by lorenzleutgeb** [comment]
the table in the bottom could be made smaller. For example, only a ranking, with no detailed skill-based results. Then we could display the list of tasks beside it.

**Comment by victorbalan** [comment]
And maybe a + button to expand it with skills. WDYT?

**Comment by lorenzleutgeb** [comment]
Yeah, or show something more detailed (result of the user). I'd say give it a try and start implementing ...

**Comment by lorenzleutgeb** [comment]
#52 implements a first version.

**Comment by pbochis** [comment]
#58 implements the skill Chart
